packages/yoroi-extension/app/connector/components/signin/cardano/UtxoDetails.js

```diff
@@ -106,6 +106,12 @@ class CardanoUtxoDetails extends Component<Props> {
     addressIndex: number,
     transform?: BigNumber => BigNumber,
   |}) => Node = request => {
+    if (request.address == null) return null;
+
+    const addressValue = request.address.value ?? request.address.values;
```

Comment:
@nistadev , not clear what is happening here, it seems like the field `. values` does not exist in any of the types the `.address` can be. There's a flow error on this line, plz check and fix.

Reply:
Hmmm weird, since our dapp connector returns `.values` for the address, maybe I'm wrong and it's a typo but examples worked well 🤔